include/aspect/utilities.h

```diff
@@ -95,7 +95,24 @@ namespace aspect
      */
     template <int dim>
     Point<dim>
-    cartesian_coordinates(const std_cxx11::array<double,dim> &scoord);
+    spherical_to_cartesian_coordinates(const std_cxx11::array<double,dim> &scoord);
+
+    /**
+     * Returns ellispoidal coordinates of a cartesian point. The returned array
+     * is filled with phi, theta and radius.
+     *
+     */
+    template <int dim>
+    std_cxx11::array<double,3>
+    cartesian_to_ellipsoidal_coordinates(const Point<3> &position, const double semi_major_axis_a, const double eccentricity);
+
+    /**
+     * Return the cartesian point of a ellispoidal position defined by phi,
+     * phi and radius.
```

Comment:
In the elliptic functions, it will be necessary to document the alignment of the major axis and the ellipticity. Presumably, they are somehow aligned with the coordinate system. But how?